example/lib/main.dart

```diff
@@ -17,17 +17,17 @@ class MyApp extends StatelessWidget {
 }
 
 class MyHomePage extends StatefulWidget {
-  MyHomePage({Key key, this.title}) : super(key: key);
+  MyHomePage({Key? key, this.title}) : super(key: key);
 
 
-  final String title;
+  final String? title;
```

Comment:
Doesn't seem like a proper application of null safety. Make arguments non nullable

Reply:
sorry, I don't see this message.... I will try to do better than this PR.